coffee-backend/src/main/kotlin/chat/letscoffee/security/security/UserPrincipal.kt

```diff
@@ -0,0 +1,70 @@
+package chat.letscoffee.security.security
+
+import chat.letscoffee.security.model.User
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import org.springframework.security.core.userdetails.UserDetails
+import org.springframework.security.oauth2.core.user.OAuth2User
+
+class UserPrincipal(val id: Long?, val email: String, private val password: String?, private val authorities: Collection<GrantedAuthority>) : OAuth2User, UserDetails {
+    private var attributes: Map<String, Any>? = null
+
+    override fun getPassword(): String? {
```

Comment:
Remove `?`

Reply:
A user doesn't have password is used with Social Login